docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1814,3 +1814,32 @@ The connector supports redirection from Iceberg tables to Hive tables with the
 
 The connector supports configuring and using [file system
 caching](/object-storage/file-system-cache).
+
+### Memory file system cache
+
+The connector supports configuring and using memory file system caching. To
+enable memory file system caching, you must set the
+`iceberg.metadata-cache.enabled` configuration property to `true`.
```

Comment:
This is already `true` by default, so we should rather say set this to `false` to disable it